### PR TITLE
define paths to MerkleProof2019, ChainedProof2021

### PR DESCRIPTION
**Pull request by lemoustachiste**
Please review @msporny

### CODE REVIEW

**Review by msporny — APPROVED** [review]
Small suggestion in review comments, otherwise LGTM.

Waiting for feedback from submitter before merging.

File: security/.htaccess
Comment (lemoustachiste) [reply security/.htaccess:89]: I am not 100% knowledgeable with jsonld extensions but it looks like I can open them in most browsers (didn't try Edge as not on a windows machine, but I think it's based of Chromium so it should work). It also works with curl and in python context too.

I have at times seen issues when trying to read a distant context file as json, issue that I have mitigated with using the `documentLoader` pattern. But I've also had it with `jsonld` files, as mitigated in this earlier version: https://github.com/blockchain-certificates/cert-verifier-js/blob/fd248e8db65aa3fe3d20d9f60c3e0979a268fea2/src/constants/contexts/preloadedContexts.ts#L42

I find problems in JS though, as the jsonld file cannot be loaded by either Jest or Rollup. I am talking about low level consuming the data from the jsonld file (when mapping for the `documentLoader` for instance), as I would normally do from a JSON file. I assume I'd need plugins for those cases but that's not something I have ever done before. 

I could also go a duplicate route (maintaining both json and jsonld file formats), but we all know that's not optimal. 

If you are loose on the suggestion, I would rather keep them as json files, but if consensus goes towards `jsonld` file format I'll abide.

Comment (OR13) [reply security/.htaccess:89]: ^ +1... regarding the pain of `.jsonld`, I have plenty of pain trying too `require` or use it when a plain json file would have worked... but this is usually a local file issue, not an http resolution one.

Github sets the accept header correctly for `.jsonld` to `content-type: application/ld+json`.

Your current web server config is returning: `content-type: application/json; charset=utf-8`.

**Review by OR13 — APPROVED** [review]

**Pull request merged**
